backend/src/main/controller/EnumController.ts

```diff
@@ -0,0 +1,64 @@
+import { DiscountType } from '../data/DiscountType';
+import { PromotionCategory } from '../data/PromotionCategory';
+import { CuisineType } from '../data/CuisineType';
+import { NextFunction, Request, Response } from 'express';
+import Joi from 'joi';
+
+enum SupportedEnums {
+  DiscountType = 'DiscountType',
+  PromotionCategory = 'PromotionCategory',
+  CuisineType = 'CuisineType',
+}
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
Feel free to remove this linting rule if it's been more annoying than helpful 😛

Reply:
hehe okay thank you 😅